packages/playground/src/views/sshkey_view.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <v-card>
+    <v-alert v-if="hasEnoughBalance" type="info" variant="tonal" class="ma-4">
+      Updating or generating SSH key will cost you up to 0.01 TFT
+    </v-alert>
+    <v-alert v-else-if="balance" type="warning" variant="tonal" class="ma-4">
+      Your balance is not enough to {{ profileManager.profile?.ssh ? "update the" : `generate a` }} SSH key; please make
```

Comment:
can u change `;` before `please` to `,`

Reply:
I thought your comment was resolved, I fixed it in another commit https://github.com/threefoldtech/tfgrid-sdk-ts/commit/5b50681261cf6e97e63a9dc4eac5535210aaae9b